fe1-web/src/core/network/NetworkManager.ts

```diff
@@ -162,14 +165,16 @@ class NetworkManager {
     }
   }
 
-  public disconnectFrom(address: string, intentional = true): void {
+  public disconnectFrom(address: string, intentional = true): Number {
```

Comment:
Maybe add a tsdoc describing the return value? Also did you intentionally chose to return `Number` and not `number`? (https://stackoverflow.com/a/67155149)

Reply:
The tsdoc can be useful to have.
No, it was not intentional and I just learned about the difference between them from the link you shared. Thank you. Will be switching to `number` instead.